netty/src/main/java/io/grpc/transport/netty/NettyServerHandler.java

```diff
@@ -287,6 +289,12 @@ private void sendResponseHeaders(ChannelHandlerContext ctx, SendResponseHeadersC
     encoder().writeHeaders(ctx, cmd.streamId(), cmd.headers(), 0, cmd.endOfStream(), promise);
   }
 
+  private void cancelStream(ChannelHandlerContext ctx, CancelServerStreamCommand cmd,
+      ChannelPromise promise) {
+    serverStream(requireHttp2Stream(cmd.streamId())).abortStream(cmd.reason(), false);
```

Comment:
Doh! I had the wrong line for that [double-checking for deadline](https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/ClientCallImpl.java#L271). I just fixed the link.